service/inventory/impl/addItemBrand.go

```diff
@@ -0,0 +1,47 @@
+package impl
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/calvinkmts/expert-pancake/engine/errors"
+	"github.com/calvinkmts/expert-pancake/engine/httpHandler"
+	db "github.com/expert-pancake/service/inventory/db/sqlc"
+	"github.com/expert-pancake/service/inventory/model"
+	uuid "github.com/satori/go.uuid"
+)
+
+func (a inventoryService) AddItemBrand(w http.ResponseWriter, r *http.Request) error {
+
+	var req model.AddBrandRequest
```

Comment:
ini request and repsonse nya tolong disamain dengan endpoint nya... kalau AddItemBrand brati AddItemBrandRequest dan AddItemBrandResponse

Reply:
add Item Brand itu punya e mas @verrellg97  sih mas..

Reply:
ok wait